protocols/dropbox.md

```diff
@@ -23,7 +23,9 @@ You can connect to multiple accounts at the same time. Create a new bookmark for
 
 ### OAuth Reset
 
-If you have accidentally logged in with the wrong Dropbox Account or want to change the login of the Dropbox bookmark you can reset the OAuth token by deleting the entries related to `https://Dropbox (user)@api.dropboxapi.com` out of the *Windows Credential Manager* or on macOS out of *Keychain Access.app*.
+If you have accidentally logged in with the wrong Dropbox Account or want to change the login of the Dropbox bookmark delete the current bookmark and create a new one to start a new authentication flow.
+
+Alternatively, you can reset the OAuth token by deleting the entries related to `duck:dropbox?user=(user)` out of the *Windows Credential Manager* or on macOS the entries related to `https://Dropbox (user)@api.dropboxapi.com` out of *Keychain Access.app*.
```

Comment:
On macOS the entries can be found by _Account_ named `Dropbox (Email) OAuth2 Access Token` or _Name_ `api.dropboxapi.com`.